cabal-install/cabal-install.cabal

```diff
@@ -214,12 +217,15 @@ executable cabal
     -- NOTE: when updating build-depends, don't forget to update version regexps
     -- in bootstrap.sh.
     build-depends:
+        async      >= 2.0      && < 3,
```

Comment:
Added `async` (see https://github.com/haskell/cabal/pull/3244). `cryptohash` and `byteable` are already there, since `hackage-security` depends on them.